pom.xml

```diff
@@ -37,6 +37,7 @@
     <raml-module-builder.version>29.0.1</raml-module-builder.version>
     <vertx.version>3.8.1</vertx.version>
     <aspectj.version>1.9.1</aspectj.version>
+    <folio-service-tools.version>1.3.1-SNAPSHOT</folio-service-tools.version>
```

Comment:
current snapshot version of the library is 1.3.2-SNAPSHOT, v1.3.1 has been released last month
Are you sure you need 1.3.1-SNAPSHOT?

Reply:
Thanks, yes, it should be 1.3.2-SNAPSHOT